server/src/main/java/com/linecorp/centraldogma/server/internal/api/auth/ApplicationTokenAuthorizer.java

```diff
@@ -65,12 +65,13 @@ public CompletionStage<Boolean> authorize(ServiceRequestContext ctx, HttpRequest
         tokenLookupFunc.apply(token.accessToken())
                        .thenAccept(appToken -> {
                            if (appToken != null && appToken.isActive()) {
-                               final StringBuilder login = new StringBuilder(appToken.appId());
+                               final String appId = appToken.appId();
+                               final StringBuilder login = new StringBuilder(appId);
                                final SocketAddress ra = ctx.remoteAddress();
                                if (ra instanceof InetSocketAddress) {
                                    login.append('@').append(((InetSocketAddress) ra).getHostString());
                                }
-
+                               ctx.logBuilder().authenticatedUser(appId);
```

Comment:
How about adding a prefix to easily distinguish `appId` and `username`?
`user/{username}`, `app/{appId}`?
```suggestion
                               ctx.logBuilder().authenticatedUser("app/" + appId);
```

Reply:
That's a good suggestion!